src/config/jwt.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  /*
+   *--------------------------------------------------------------------------
+   * JWT Secret
+   *--------------------------------------------------------------------------
+   *
+   * Here your jwt secret key. In order to verify a token injected in
+   * "Authorization" header, we should have way to decrypt that token
+   * and validate things.
+   *
+   */
+
+  secret: 'YOUR-JWT-TOKEN-HERE',
+
+  /*
+   *--------------------------------------------------------------------------
+   * Issuer
+   *--------------------------------------------------------------------------
+   *
+   * Here your issuer validation, if you would like to validate the issuer
+   * you could add the lists of domains below and activate the validation.
+   *
+   */
+
+  iss: {
+    validate: false,
+    data: ['api.domain.com'],
```

Comment:
Should be configurable per environment. i.e;

```
process.env.JWT_ISS_SHOULD_VALIDATE=
process.env.JWT_ISS_DOMAIN=
```

Reply:
Applied above